worlds/cccharles/Locations.py

```diff
@@ -0,0 +1,914 @@
+from BaseClasses import Location
+from .BaseID import base_id
+
+
+class CCCharlesLocation(Location):
+    game: str = "Choo-Choo Charles"
```

Comment:
```suggestion
    game = "Choo-Choo Charles"
```
Slightly more consistent with other places to not have the probably unnecessary annotation here.